onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -1826,6 +1984,9 @@ including arg name, arg type (contains both type and shape).)pbdoc")
       .value("kNextPowerOfTwo", onnxruntime::ArenaExtendStrategy::kNextPowerOfTwo)
       .value("kSameAsRequested", onnxruntime::ArenaExtendStrategy::kSameAsRequested)
       .export_values();
+
+  py::class_<AsyncFuture>(m, "future")
```

Comment:
A docstring for the class would be helpful as well.

Reply:
Code outdated, but thanks!